lib/barbeque_client/executor.rb

```diff
@@ -8,9 +8,19 @@ class Executor
     # @param [String] queue_name - barbeque's job_queues.name
     def initialize(job:, message:, message_id:, queue_name:)
       @job        = job
-      @message    = JSON.load(message)
```

Comment:
[just my feeling, no objection] I'd simply change to `@message = Array(JSON.load(message))` or `@message = Array.wrap(JSON.load(message))` because branching is not utilized for comments.